### DISCUSSION TITLE
Cross Validation Metrics

### DISCUSSION THREAD

**Discussion by jmrichardson** [microsoft/FLAML#662] [Ideas]
Hi,  

It appears the metrics for cross validation by fold are aggregated (averaged) and stored in the log file.  Is there a way to also log each fold metric?  It would also be great to log the standard deviation to account for model robustness.

**Comment by sonichi** [comment]
Good suggestion. @skzhang1 could you follow up with @jmrichardson on how to do it?

@jmrichardson I'm also curious about your use case. Can we chat on [gitter](https://gitter.im/FLAMLer/community)?

**Comment by skzhang1** [comment]
@sonich Sure, I have posted a PR to support a customized cross-validation strategy. 
@jmrichardson Thanks for your suggestion. We will update the main branch to support this new feature after checking.
